pkg/kubelet/rkt/container_id.go

```diff
@@ -41,6 +42,14 @@ func buildContainerID(c *containerID) kubecontainer.ContainerID {
 	}
 }
 
+func apiPodToNetPluginContainerID(pod *api.Pod) kubecontainer.ContainerID {
+	return buildContainerID(&containerID{string(pod.UID), pod.Name})
```

Comment:
Why you use rkt's buildContainerID here? This function generates the runtime container ID, not the API pod UID
I think you want to use the API pod's UID as the network namespace's name, right?